nicknamer.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+# noinspection PyPackageRequirements
+from discord import Member, Client
+# noinspection PyPackageRequirements
+from discord.ext.commands import Context, Bot
+
+from keep_alive import keep_alive
+
+TOKEN = os.getenv("DISCORD_TOKEN", None)
```

Comment:
This can just be `os.getenv("DISCORD_TOKEN")`, the default behavior is to write None if an env var is not found.